src/mparticle-instance-manager.ts

```diff
@@ -17,21 +21,26 @@ if (!Array.prototype.filter) {
     Array.prototype.filter = Polyfill.filter;
 }
 
+// https://go.mparticle.com/work/SQDSDKS-6768
 if (!Array.isArray) {
+    // @ts-ignore
     Array.prototype.isArray = Polyfill.isArray;
 }
 
-function mParticle() {
-    var self = this;
+// Question: Should we just change this to mParticleInsanceManager?
```

Comment:
I think that's fine.  we end up instantiating it below, so this is the correct name
```suggestion
// Question: Should we just change this to mParticleInstanceManager?
```